plugins/services/src/js/components/forms/HealthChecksFormSection.js

```diff
@@ -39,7 +71,9 @@ class HealthChecksFormSection extends Component {
             <FormGroup
               className="column-3"
               showError={Boolean(errors.gracePeriodSeconds)}>
-              <FieldLabel>Grace Period (s)</FieldLabel>
+              <FieldLabel helpText={gracePeriodHelpText}>
```

Comment:
Yeah, I still don't feel this is the declarative approach. @jfurrow has worked on this I think that is a superior solution: https://github.com/dcos/dcos-ui/pull/1887/files#diff-82fc08fe0a4224982c1e31ce7424c416R97

What do you say to reverting to what we do elsewhere and let that PR handle it?